app/assets/javascripts/discourse/controllers/reorder-categories.js.es6

```diff
@@ -38,29 +38,22 @@ export default Ember.Controller.extend(ModalFunctionality, Ember.Evented, {
     return anyChanged;
   }.property('categoriesBuffered.@each.hasBufferedChanges'),
 
-  saveDisabled: Ember.computed.alias('showApplyAll'),
+  @computed('showApplyAll', 'showFixIndices')
+  saveDisabled(a, b) {
```

Comment:
`a || b` isn't clear at all. I'd rename the variables to be what they represent or just use the `Ember.computed.or`CP macro